### PR TITLE
Clean up from details toggle ticket

### PR DESCRIPTION
**Pull request by lindsayrmaher**
### What's this PR do?

This pull request does some clean up based on the conversation from #1872 !

### How should this be reviewed?

👀 

### Any background context you want to provide?

Wanted to follow through on the suggestions to more explicitly define what props we could pass to `MenuCarat` and get rid of unnecessary variables.

### Relevant tickets

References [Pivotal #169929291](https://www.pivotaltracker.com/story/show/169929291).

### Checklist

- [x] This PR has been added to the relevant Pivotal card.
- [ ] Documentation added for new features/changed endpoints.
- [ ] Added screenshots of front-end changes on small, medium, and large screens.
- [ ] Added appropriate feature/unit tests.

### CODE REVIEW

File: resources/assets/components/blocks/ScholarshipInfoBlock/ScholarshipInfoBlock.js [hunk 1]
```diff
@@ -224,7 +220,9 @@ const ScholarshipInfoBlock = ({
               onClick={toggleHiddenInfo}
             >
               <p className="text-sm font-bold pr-2">{`${detailsLabel} Details`}</p>
-              <MenuCarat css={drawerOpen ? caratRotate : null} />
+              <MenuCarat
+                cssStyles={drawerOpen ? `transform: rotate(180deg);` : null}
```
Comment (DFurnes) [inline resources/assets/components/blocks/ScholarshipInfoBlock/ScholarshipInfoBlock.js]: Would this make sense to provide as a `` css`…` `` so we get syntax highlighting and such? That seems to be suggested in Emotion's [Composition docs](https://emotion.sh/docs/composition).

File: resources/assets/components/artifacts/MenuCarat/MenuCarat.js [hunk 2]
```diff
@@ -28,12 +28,14 @@ const MenuCarat = ({ color, height, width, ...props }) => (
 
 MenuCarat.propTypes = {
   color: PropTypes.string,
+  cssStyles: PropTypes.string,
```
Comment (DFurnes) [inline resources/assets/components/artifacts/MenuCarat/MenuCarat.js]: If we do send a `css` template string, there might be better (maybe `instanceOf`?) prop-type to use here!

File: resources/assets/components/blocks/ScholarshipInfoBlock/ScholarshipInfoBlock.js (hunk 1 above)
Comment (lindsayrmaher) [reply resources/assets/components/blocks/ScholarshipInfoBlock/ScholarshipInfoBlock.js]: Hmmmmm like this! even more clear that it's emotion specific.

File: resources/assets/components/artifacts/MenuCarat/MenuCarat.js (hunk 2 above)
Comment (lindsayrmaher) [reply resources/assets/components/artifacts/MenuCarat/MenuCarat.js]: well if we do use the css template string it gets interpreted as an obj, so maybe better to update to an obj?

Comment (DFurnes) [reply resources/assets/components/artifacts/MenuCarat/MenuCarat.js]: Ah yeah, if it's just a plain object then that'll do it! 👍

**Pull request merged**